internal/pkg/model/naming.go

```diff
@@ -153,23 +153,33 @@ func (n Naming) ConfigPath(parentPath string, component *Component, config *Conf
 		panic(fmt.Errorf(`config "%s" parent path cannot be empty"`, config))
 	}
 
+	// Get parent in the local filesystem
+	parentKey, err := config.ParentKey()
+	if err != nil {
+		panic(err)
+	}
+
 	// Shared code is handled differently
 	var template, targetComponentId string
-	if component.IsSharedCode() {
-		// Get target component ID for shared code config
-		if config.Content == nil {
-			panic(fmt.Errorf(`shared code config "%s" must have set key "%s"`, config.Desc(), ShareCodeTargetComponentKey))
-		}
-		targetComponentIdRaw, found := config.Content.Get(ShareCodeTargetComponentKey)
-		if !found {
-			panic(fmt.Errorf(`shared code config "%s" must have set key "%s"`, config.Desc(), ShareCodeTargetComponentKey))
+	if parentKey.Kind().IsBranch() {
```

Comment:
Pridal som tu zistovanie toho, akeho typu je `parent` (co sa tyka adresarovej struktury).

Variables budu na disku sucastou configu, ktory ich pouziva, ...  a teda potrebujem pri generovani nazvu vediet, ci je parent `branch` alebo `config`.

Tu to samo o sebe velky zmysel nedava, ale nechcem som poslat prilis velky PR. 
Pokracovanie v dalsom, aktualnu funkcionalitu to neovplyvnuje.